baseplate/experiments/__init__.py

```diff
@@ -189,8 +201,36 @@ def variant(self, name, user=None, bucketing_event_override=None,
 
         return variant
 
+    def expose(self, experiment_name, variant_name, user=None, **kwargs):
+
+        experiment = self._get_experiment(experiment_name)
+
+        if experiment is None:
+            return
+
+        inputs = dict(kwargs)
+
+        if user:
+            if user.is_logged_in:
+                inputs["user_id"] = user.id
+                inputs["user_roles"] = user.roles
+            else:
+                inputs["user_id"] = user.loid
+            inputs["logged_in"] = user.is_logged_in
+            inputs['cookie_created_timestamp'] = user.event_fields().get('cookie_created')
+
+        self._event_logger.log(
+            experiment=experiment,
+            variant=variant_name,
+            user_id=inputs.get('user_id'),
+            logged_in=inputs.get('logged_in'),
+            cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
+            app_name=inputs.get('app_name'),
+            exposed=True,
```

Comment:
Controlling this with a boolean flag seems pretty inflexible, maybe we want to send a string value `event_type` that can be`'bucket'` or `'exposure'`?

Reply:
enum would be nicer than strings if it's a multi-option but finite set

Reply:
updated